src/lib/middleware/errorHandler.ts

```diff
@@ -47,3 +48,23 @@ export function errorHandlerMiddleware(
 
   res.status(code).render(file, { code, detail, message })
 }
+
+export function setupErrorHandling(app) {
```

Comment:
I know there was a comment in the original file to look into moving this, however, this feels far less intuitive than leaving it as it was.

Reply:
Why? There's a function called `setupErroHandling` right where this block of code would be, only now its been moved to the `errorHandler` file, which actually handles errors. So its logical and hides away unnecesary implementation details by co-locating it with its sibling.

Reply:
With that raven line needing to be first, will move this stuff back in